DEPLOY.md

```diff
@@ -0,0 +1,51 @@
+# Cassandra Diagnostics Releasing
+
+This document will explain what to do when new version is ready for release.
+
+## Versioning
+
+We are versioning Cassandra Diagnostics with major.minor.patch versions. If changes included in new release are minor fixes and improvements we increase patch version. If subject of new release is new feature we increase minor version. If we changed extensively (adding new module, change in project structure) we increase major version and do release.
+
+## Releasing
+
+1. Increase new release version
+
+First step is to set new version in parent pom, and in all pom files in each submodule. This guide will follow release from 1.1.1. to 1.1.2. but it can be applyed to any new release we make.
+
+```
+mvn versions:set -DnewVersion=1.1.1
```

Comment:
If we are following release from 1.1.1 to 1.1.2 then new version should be 1.1.2